data/language/dutch.txt

```diff
@@ -2382,19 +2382,19 @@ STR_2380    :{SMALLFONT}{BLACK}Kleiner stuk water aanpassen
 STR_2381    :{SMALLFONT}{BLACK}Groter stuk water aanpassen
 STR_2382    :Land
 STR_2383    :Water
-STR_2384    :{WINDOW_COLOUR_2}Je doel:
+STR_2384    :{WINDOW_COLOUR_2}Je doelstelling:
```

Comment:
Ik ben destijds ook akkoord gegaan met het veranderen van 'segment' in 'gedeelte'. Hoe dan ook, ik ben benieuwd naar de mening van de andere vertalers over 'doel' vs. 'doelstelling'.